pkg/tfvars/powervs/powervs.go

```diff
@@ -76,7 +74,7 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 		VPCZone:              vpcZone,
 		PowerVSResourceGroup: sources.PowerVSResourceGroup,
 		CISInstanceCRN:       sources.CISInstanceCRN,
-		ImageName:            *masterConfig.Image.Name,
+		ImageBucketFileName:  *masterConfig.Image.Name,
```

Comment:
`masterConfig.Image` should be the boot image we create, and not the ova. so this should stay as `ImageName`, and we'll need a new field for the bucket object name (ImageBucketFileName is fine). `ImageName` will be the same as `pi_image_name` (`"${var.cluster_id}-boot-image"`). `ImageBucketFileName` will need to come down from the bits that Cory is working on I believe.

Reply:
Yep that's fair, will do